extras/check_code_style.sh

```diff
@@ -232,7 +239,13 @@ for f in $file_names; do
     *.py )
       echo "Check PEP8 on file $f:"
 
-      if ! pep8_result=`pep8 --first --ignore=$PEP8_IGNORES $f` ; then
+      if [[ $f =~ $EXAMPLE_DIRS ]]; then
```

Comment:
@heplesser I just found flake8 which is a wrapper around pep8 to make pep8 less religious, including turning off warnings on a file by file basis. Maybe worthwhile using?